.kitchen.yml

```diff
@@ -24,3 +25,6 @@ suites:
   - name: organizational-k8s
     driver:
       root_module_directory: test/fixtures/organizational-k8s
+  - name: single-single-account-apprunner
```

Comment:
small typo here :D 
_all the single ladies..._  👯 

not sure why CI works on this test, as it's pinned to  ` kitchen test "single-account-apprunner-aws"`
know kitchen does some regexp on test selection if its name is not quoted, but in your case it is 🤔 
i would fix name either way for the future us